packages/util-utf8/src/fromUtf8.browser.ts

```diff
@@ -0,0 +1,7 @@
+import { fromUtf8 as jsFromUtf8 } from "./pureJs";
+import { fromUtf8 as textEncoderFromUtf8 } from "./whatwgEncodingApi";
+
+declare const TextEncoder: Function | undefined;
+
+export const fromUtf8 = (input: string): Uint8Array =>
+  typeof TextEncoder === "function" ? textEncoderFromUtf8(input) : jsFromUtf8(input);
```

Comment:
TextEncoder is supported in all browsers which support ES2020
We can remove `jsFromUtf8`, and copy `textEncoderFromUtf8` inside this function

MDN: https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder#browser_compatibility
PR which bumped ES version to ES2020: https://github.com/aws/aws-sdk-js-v3/pull/4005

Reply:
fixed